### DISCUSSION TITLE
Unable to upgrade to Premium

### DISCUSSION THREAD

**Discussion by gruesomegpt** [vercel/community#6367] [Account Issue]
### Question

I am from India and tried to upgrade v0.dev to premium using Amex and HDFC card but couldnt do. It said payment failed for both. What is the workaround?

**Comment by amyegan** [comment]
Hello @gruesomegpt. If you submit an **Invoice Enquiry** case, that will get you in contact with our billing experts. 

You can find the form to submit a case at [vercel.com/help](https://vercel.com/help)
